lib/bundle/locker.rb

```diff
@@ -128,6 +130,15 @@ def mas_list
       end
     end
 
+    def whalebrew_list
+      @whalebrew_list ||= begin
+        Bundle::WhalebrewDumper.images.each_with_object({}) do |image, name_versions|
+          version = `docker image inspect #{image} --format '{{ index .RepoDigests 0 }}' | awk -F':' '{print $2}'`.chomp
```

Comment:
Can this line be shortened? Additionally, would be nice to parse this with Ruby rather than with `awk`.